include/geode/model/helpers/component_mesh_queries.h

```diff
@@ -73,6 +77,37 @@ namespace geode
         absl::optional< BlockPolyhedronFacet > opposite_polyhedron_facet;
     };
 
+    struct SurfacePolygonEdge
+    {
+        SurfacePolygonEdge() = default;
+        SurfacePolygonEdge( PolygonEdge edge_in ) : edge{ std::move( edge_in ) }
+        {
+        }
+
+        PolygonEdge edge;
+        std::array< index_t, 2 > vertices;
```

Comment:
Same reason as for BlockPolyhedronFacet: those vertices are proprietary, since they are ordered in a specific way in the arrayt (same way as the input edge in the functions using it).